app/src/main/java/com/example/tasky/agenda/presentation/util/AgendaDetailConfigProvider.kt

```diff
@@ -21,6 +20,16 @@ object AgendaTypeConfigProvider {
         }
     }
 
+    fun getDefaultReminderIntervals(): List<AgendaReminderInterval> {
+        return listOf(
+            AgendaReminderInterval(600000L, "10 minutes before"),
```

Comment:
This looks good, but it might be an even better fit for `UiText` instead of hardcoded strings. That way your reminder labels can come from string resources, stay localizable, and still be passed around cleanly without a Context

```kotlin
UiText.StringResource(R.string.reminder_10_min),
UiText.StringResource(R.string.reminder_30_min),
UiText.StringResource(R.string.reminder_1_hour),
UiText.StringResource(R.string.reminder_6_hours),
UiText.StringResource(R.string.reminder_1_day)
```